src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/service/SystemService.java

```diff
@@ -139,4 +153,23 @@ private String getExceptionMessage(String name, String url, String message) {
         sb.append(" ").append(message);
         return sb.toString();
     }
+
+    /**
```

Comment:
Very minor thing but I think Exceptions in serviceclasses have been last thing to declare.

Reply:
Moved the exception to the bottom.